### PR TITLE
Update getting-started-part-three.md

### PR DESCRIPTION
**Pull request by Trey-O**
Include note that GitHub Repo is needed, as well a CodeShip Pro Project and workaround for generating auth key in Mac.

### CODE REVIEW

File: _pro/quickstart/getting-started-part-three.md [hunk 1]
```diff
@@ -37,6 +37,12 @@ Now that you have a working CI process with a successful test, the next step is
 
 ### Required Services 
 
+#### Project linked to a Git Repository
+
+Within your SCM of choice you need to have configured the directory where you have been making these files as a Repository.
```
Comment (joesiewert) [inline _pro/quickstart/getting-started-part-three.md]: Would lowercase repository

File: _pro/quickstart/getting-started-part-three.md [hunk 2]
```diff
@@ -37,6 +37,12 @@ Now that you have a working CI process with a successful test, the next step is
 
 ### Required Services 
 
+#### Project linked to a Git Repository
+
+Within your SCM of choice you need to have configured the directory where you have been making these files as a Repository. 
+
+You also need to have linked this Repository in your SCM to a CodeShip Pro Project at [CodeShip UI](https://app.codeship.com/home)
```
Comment (joesiewert) [inline _pro/quickstart/getting-started-part-three.md]: Lowercase repository and project - would also link to the project page instead (https://app.codeship.com/projects)

File: _pro/quickstart/getting-started-part-three.md (hunk 1 above)
Comment (bwalding) [inline _pro/quickstart/getting-started-part-three.md:42]:
```suggestion
Within your SCM of choice you need to have configured the directory where you have been making these files as a repository. 
```

File: _pro/quickstart/getting-started-part-three.md (hunk 2 above)
Comment (bwalding) [inline _pro/quickstart/getting-started-part-three.md:44]:
```suggestion
You also need to have linked this repository in your SCM to a CodeShip Pro project at [CodeShip UI](https://app.codeship.com/home)
```

File: _pro/quickstart/getting-started-part-three.md [hunk 3]
```diff
@@ -117,6 +123,8 @@ cat ${HOME}/.docker/config.json
 The newer versions of Docker have changed to store credentials in the MacOS keychain rather than in a configuration file.
 
 To get the appropriate authentication file on MacOS, you will need to remove the `credsStore` line from Docker's `config.json` to disable Keychain storage, re-run `docker login` and then use the values it then generates in your updated `dockercfg` as shown above.
+
+**NOTE:** If this method to retrieve your authentication does not work you may manually generate your authentication key by running the following command in your terminal: `echo -n 'DOCKER_HUB_USERNAME:DOCKER_HUB_PASSWORD' | base64` and then copy this encrypted auth key to your `dockercfg` file.
```
Comment (bwalding) [inline _pro/quickstart/getting-started-part-three.md:127]:
```suggestion
**NOTE:** If this method to retrieve your authentication does not work, you may manually generate your authentication key by running the following command in your terminal: `echo -n 'DOCKER_HUB_USERNAME:DOCKER_HUB_PASSWORD' | base64` and then copy this encrypted authentication key to your `${HOME}/.docker/config.json` file. 
```

**Comment by bwalding** [comment]
@joesiewert Are we good to go on merging this? Want to close out / finish / destroy as many long running PRs as I can!

**Comment by joesiewert** [comment]
This is old and don't think it matters anymore.

**Pull request closed without merging**